usermanager/src/main/java/org/pesmypetcare/usermanager/clients/pet/PetManagerClient.java

```diff
@@ -168,8 +170,8 @@ public Object getSimpleField(String accessToken, String username, String petName
         PetData.checkSimpleField(field);
         taskManager = taskManager.resetTaskManager();
         taskManager.setTaskId(GET);
-        StringBuilder response = taskManager.execute(BASE_URL + PETS_PATH + username + SLASH
-                + petName + "/simple/" + field, accessToken).get();
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/simple/" + field, accessToken).get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/simple/" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561031)